shared/stories/__tests__/__snapshots__/Storyshots.test.js.snap

```diff
@@ -260150,96 +260150,6 @@ Array [
               >
                 Keybase is enabled in your Finder.
               </span>
-              <div
```

Comment:
have you made sure this story change is intentional?

Reply:
I remember noticing this too, but checked all relative changes and thought it was fine so I just left it assuming master storybook inconsistency. I just checked again, and guess what ... I forgot to update the storybook prop-providers to de-immutable-ize! Thanks for calling it out.